qa/L0_perf_analyzer_report/test.sh

```diff
@@ -166,7 +160,7 @@ set -e
 
 # Cleanup
 kill $SERVER_PID
-wait $SERVER_PID
+
```

Comment:
@lkomali doesn't this `wait $SERVER_PID` need to be added back after you fixed the root cause with inflight request counter?